src/Components/Publishing/Article.tsx

```diff
@@ -45,7 +59,42 @@ export interface ArticleProps {
     dispatch: data => Events.postEvent(data),
   }
 )
-export class Article extends React.Component<ArticleProps> {
+export class Article extends React.Component<ArticleProps, State> {
+  state = {
+    showCtaBanner: false,
+  }
+  lastScrollPosition: number = 0
+
+  handleScroll() {
+    const newScrollPosition = window.scrollY
+    let showCtaBanner = this.state.showCtaBanner
+
+    if (newScrollPosition <= this.lastScrollPosition) {
+      // scrolling up the page
+      if (this.state.showCtaBanner) {
+        showCtaBanner = false
+      }
+    } else {
+      // scrolling down the page
+      if (!this.state.showCtaBanner) {
+        showCtaBanner = true
+      }
+    }
+
+    if (this.state.showCtaBanner !== showCtaBanner) {
+      this.setState({ showCtaBanner })
+    }
+    this.lastScrollPosition = newScrollPosition
+  }
+
+  componentDidMount() {
+    if (window) {
+      window.addEventListener("scroll", e => {
+        this.handleScroll()
+      })
```

Comment:
Might be a good idea to debounce `handleScroll` here so we aren't calling every pixel.